metaflow/__init__.py

```diff
@@ -100,6 +100,12 @@ class and related decorators.
 # current runtime singleton
 from .metaflow_current import current
 
+# system monitor runtime singleton
```

Comment:
I would add a comment like (for internal metaflow use only). It's kind of explicit from the name but we can be extra explicit :)